src/main/resources/com/google/api/codegen/php/client_impl.snip

```diff
@@ -348,24 +361,42 @@
      *           path to a JSON file, or a PHP array containing the decoded JSON data.
      *           By default this settings points to the default client config file, which is provided
      *           in the resources folder.
-     *     @@type string|TransportInterface $transport
-     *           The transport used for executing network requests. May be either the string `rest`
-     *           or `grpc`. Defaults to `grpc` if gRPC support is detected on the system.
-     *           *Advanced usage*: Additionally, it is possible to pass in an already instantiated
-     *           {@@see \Google\ApiCore\Transport\TransportInterface} object. Note that when this
-     *           object is provided, any settings in $transportConfig, and any $serviceAddress
-     *           setting, will be ignored.
-     *     @@type array $transportConfig
-     *           Configuration options that will be used to construct the transport. Options for
-     *           each supported transport type should be passed in a key for that transport. For
-     *           example:
-     *           $transportConfig = [
-     *               'grpc' => [...],
-     *               'rest' => [...]
-     *           ];
-     *           See the {@@see \Google\ApiCore\Transport\GrpcTransport::build()} and
-     *           {@@see \Google\ApiCore\Transport\RestTransport::build()} methods for the
-     *           supported options.
+     @if xapiClass.isRestOnlyTransport
+         *     @@type string|TransportInterface $transport
+         *           The transport used for executing network requests. Defaults to `rest`.
```

Comment:
given `RestOnlyTransport`, should this `Defaults to 'rest'` be `At the moment, only supports 'rest'`?

Reply:
I like that better!